### PR TITLE
Replace println! with liblog

### LINKED ISSUE #23: Add proper logging
`liblog` has been brought up by @nrc.

### PR DESCRIPTION
**Pull request by Mark-Simulacrum**
printlns were converted to reasonable log levels, but no new logging was
added (other than printing a message on server start since there's no
longer a wall of text by default).

More detailed logging is out-of-scope, and can be introduced as the codebase is worked on/later.

Fixes #23.

### CODE REVIEW

File: rs-backend/src/main.rs [hunk 1]
```diff
@@ -7,11 +7,17 @@
...
 extern crate rustc_perf;
 
 use rustc_perf::{load, server, util};
 
 fn main() {
+    env_logger::init().unwrap();
+
     let data = load::InputData::from_fs(&util::get_repo_path().unwrap()).unwrap();
+
+    println!("Starting server!");
```
Comment (nrc) [inline rs-backend/src/main.rs]: should this be info?

Comment (Mark-Simulacrum) [inline rs-backend/src/main.rs]: I wanted to print when we are starting the server even when no logging is explicitly enabled. info is nice, but is less than warn and then we're back to wall of output.

Comment (nrc) [inline rs-backend/src/main.rs]: I usually prefer that silence => success, but I agree it is a useful notification

**Comment by nrc** [comment]
thanks!

File: rs-backend/src/main.rs (hunk 1 above)
Comment (Mark-Simulacrum) [inline rs-backend/src/main.rs]: I prefer silence being success too, but due to the long boot up, I've found it's nice to have the "finished" notification (I see ~2 minute long loads).

**Pull request merged**